webtransport/quic/client-indication.any.js

```diff
@@ -0,0 +1,32 @@
+// META: quic=true
+// META: script=/common/get-host-info.sub.js
+
+const PORT = 8983;
+const HOST = get_host_info().ORIGINAL_HOST;
+const ORIGIN = get_host_info().ORIGIN;
+const BASE = `quic-transport://${HOST:${PORT}/handlers`;
+
+promise_test(async (test) => {
+  function onClosed() {
+    assert_unreached('The closed promise should be ' +
+                     'fulfilled or rejected after getting a PASS signal.');
+  }
+  t = new QuicTransport(`${BASE}/client-indication.quic.py?origin=${ORIGIN}`);
+  t.closed.then(test.step_func(onClosed), test.step_func(onClosed));
+
+  const streams = t.receiveStreams();
+  let {done, value} = await streams.getReader().read();
```

Comment:
`{done, value}` could be const here and also below (because it is in a nested scope).

Reply:
Done.